src/middlewared/middlewared/alembic/versions/23.10/2023-01-11_14-53_add_apps_metrics_server.py

```diff
@@ -0,0 +1,25 @@
+"""Add apps metrics server
+
+Revision ID: 1896dbab6040
+Revises: 60de23d5cd17
+Create Date: 2023-01-11 14:53:38.702138+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '1896dbab6040'
+down_revision = '60de23d5cd17'
```

Comment:
Can you please update down revision ?

Reply:
Also probably let's rename the migration file name as well to reflect latest date while we are at it as it would keep the nice order in the order they are merged.

Reply:
No idea how, this is what it is.
This is byond my skill to fix without a test VM.